tests/test-setup.sh

```diff
@@ -0,0 +1,52 @@
+set -eu
```

Comment:
where do we use this file?

Reply:
I forgot to delete this file, deleting it now